ide-support/revsaveasstandalone.livecodescript

```diff
@@ -2462,23 +2462,14 @@ command revRedirectMacOSResources pStandalonePath
    create folder (tOldRFolder & slash & "../Resources")
    create folder (tOldRFolder & slash & "../Resources/_MacOS")
    
-   local tOldFolder
-   put the folder into tOldFolder
-   
    revRedirectMacOSResourcesRecurse tOldRFolder, tNewRFolder
-   
-   set the folder to tOldFolder
 end revRedirectMacOSResources
 
 command revRedirectMacOSResourcesRecurse pSourceFolder, pTargetFolder
-   set the folder to pSourceFolder
    create folder pTargetFolder
    
    local tFiles
-   repeat for each line tFile in the files
-      if tFile begins with "." then
```

Comment:
I think this line was there for the dreaded ".DS_STORE" file, so it might need to stay in (although I guess not a big deal to copy it).